tests/10_ephemeral.test.mjs

```diff
@@ -0,0 +1,223 @@
+import { restClient, validateErrorResponse } from './utils.mjs'
+import { describe, it } from 'node:test'
+import { strict as assert } from 'node:assert'
+import { pkg, corePkg } from '../api/tests/json-loader.mjs'
+import { errors } from '../api/src/errors.mjs'
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const api = restClient(`https://${process.env.MORIO_TEST_HOST}/-/api`)
+
+describe('Ephemeral API: Status Routes', () => {
+  /*
```

Comment:
I have included this example response in my original tests, but I think it's de-facto impossible to maintain.
So I would suggest removing it.

Reply:
I've removed it.